src/OpenTelemetry/Logs/OpenTelemetryLoggerProvider.cs

```diff
@@ -112,6 +112,12 @@ public ILogger CreateLogger(string categoryName)
             return logger;
         }
 
+        /// <summary>
+        /// Create a <see cref="LogEmitter"/>.
+        /// </summary>
+        /// <returns><see cref="LogEmitter"/>.</returns>
+        public LogEmitter CreateEmitter() => new(this);
```

Comment:
Do you consider LogEmitter as a pub-sub model? (similar like ActivitySource - which can be subscribed by multiple listeners)